README.md

```diff
@@ -54,7 +54,133 @@ hasp_lvgl:
+        event:
+          "down":
+            service: cover.open_cover
+            target:
+              entity_id: "cover.cover_1"
+      - obj: "p1b5"
+        properties:
+          "text": "{{ state_attr('cover.cover_1','current_position') }}" 
+        event:
+          "down":
+            service: cover.stop_cover
+            target:
+              entity_id: "cover.cover_1"
+      - obj: "p1b6"
+        properties:
+          "text_color": "{{ '#FFFF00' if states('cover.cover_1') == 'closing' else '#FFFFFF' }}"
+          "text_opa": "{{ '80' if state_attr('cover.cover_1','current_position') == 0 else '255' }}"
+        event:
+          "down":
+            service: cover.close_cover
+            target:
+              entity_id: "cover.cover_1"
+
+      - obj: "p1b7"
+        properties:
+          "text_color": "{{ '#FFFF00' if states('cover.cover_2') == 'opening' else '#FFFFFF' }}"
+          "text_opa": "{{ '80' if state_attr('cover.cover_2','current_position') == 100 else '255' }}"
+        event:
+          "down":
+            service: cover.open_cover
+            target:
+              entity_id: "cover.cover_2"
+      - obj: "p1b8"
+        properties:
+          "text": "{{ state_attr('cover.haloszoba_kozep','current_position') }}" 
+        event:
+          "down":
+            service: cover.stop_cover
+            target:
+              entity_id: "cover.cover_2"
+      - obj: "p1b9"
+        properties:
+          "text_color": "{{ '#FFFF00' if states('cover.cover_2') == 'closing' else '#FFFFFF' }}"
+          "text_opa": "{{ '80' if state_attr('cover.cover_2','current_position') == 0 else '255' }}"
+        event:
+          "down":
+            service: cover.close_cover
+            target:
+              entity_id: "cover.cover_2"
+```
+
+### Example 2: generic thermostat control
+![kép](https://user-images.githubusercontent.com/1550668/112160012-09536300-8bea-11eb-867d-53c64894c324.png)
+<br>UI theme set to `Hasp Light` in plate's web interface.
+
+**hasp-lvgl config:** (screen size 240x320) 
+```text
+{"page":2,"comment":"---------- Page 2 ----------"}
+{"obj":"arc","id":3,"x":20,"y":75,"w":200,"h":200,"min":180,"max":250,"border_side":0,"type":0,"rotation":0,"start_angle":135,"end_angle":45,"start_angle1":135,"end_angle1":45,"value_font":28,"value_color":"#2C3E50","adjustable":"true"}
+{"obj":"dropdown","id":4,"x":75,"y":235,"w":90,"h":30,"options":"off\nheat_cool\nheat\ncool\ndry\nfan_only"}
+{"obj":"btn","id":5,"x":68,"y":162,"w":25,"h":25,"toggle":false,"text":"-","text_font":28,"align":1}
+{"obj":"btn","id":6,"x":147,"y":162,"w":25,"h":25,"toggle":false,"text":"+","text_font":28,"align":1}
+{"obj":"label","id":7,"x":60,"y":120,"w":120,"h":30,"text":"Status","align":1,"padh":50}
+```
+**hasp-lvgl-custom-component config:**
+```yaml
+      - obj: "p2b3"
+        properties:
+          "val": "{{ state_attr('climate.thermostat_1','temperature') * 10 | int }}"
```

Comment:
I know your personal use case in which you need to multiply/divide, but we should explain this to users too, or remove altogether not to cause confusion

Reply:
OK

Reply:
I wouldn't call this my own _personal use case_. In real life everyone needs to set thermostat in decimal values, at least in 0.5 degrees precision.

Reply:
Mine does not handle that precision 😓

Reply:
It's time for you to implement one :-)

Reply:
Modifications requested made.

Reply:
I read somewhere the Landon has a temperature sensor... waiting for it's support 😄